packages/openactive-integration-tests/test/helpers/request-state.js

```diff
@@ -78,11 +79,13 @@ class RequestState {
           return x =>
           (Array.isArray(x.availableChannel) && x.availableChannel.includes("https://openactive.io/OpenBookingPrepayment"))
           && x.advanceBooking != "https://openactive.io/Unavailable"
+          // @ts-expect-error startDate is not defined
           && (x.validFromBeforeStartDate && moment(startDate).subtract(moment.duration(x.validFromBeforeStartDate)).isAfter());
```

Comment:
I'm very confused by this. My current theory is that this code branch happens to never gets executed as it would surely fail if it did. I did some investigation but stopped as it was taking a while.

Same goes for the below 2 `@ ts-expect-error`s

Reply:
This is indeed a bug, and I've fixed this but not committed the code - doh! Will do in a separate PR

Reply:
@lukehesluke here's the PR for this: https://github.com/openactive/openactive-test-suite/pull/100

Reply:
Excellent